lib/vspcrow.gi

```diff
@@ -161,16 +161,24 @@ InstallOtherMethod( LeftModuleByGenerators,
     [ IsDivisionRing, IsList ],
     # ensure it ranks above the generic method
     function( F, mat )
-    local V,typ;
+    local V, typ, K, row;
 
     # filter for vector objects, not compressed FF vectors
     if not ForAll(mat,x->IsVectorObj(x) and not IsDataObjectRep(x)) then
       TryNextMethod();
     fi;
     typ:=IsAttributeStoringRep and HasIsEmpty and IsFiniteDimensional;
     if ForAll( mat, row -> IsSubset( F, BaseDomain(row) ) ) then
+      # Replace 'mat' by vector objects with base domain 'F'.
+      mat:= List( mat, v -> ChangedBaseDomain( v, F ) );
       typ:=typ and IsGaussianRowSpace;
     else
+      # Replace 'mat' by vector objects with base domain containing 'F'.
+      K:= F;
```

Comment:
Can we add a test case to exercise this code path? Codecov says it currently isn't covered

Reply:
No.
The point is the `IsNonGaussianRowsSpace` filter which gets set without the necessary code for the `IsVectorObj` case.
The `else` branch did not work up to now because of this problem, and still does not work,
the aim of the proposed change is to provide at least suitable vector objects.
I will mention this in the `FIXME` comment below.